### DISCUSSION TITLE
Indicate repository owner and possible maintainers in Issue discussion

### DISCUSSION THREAD

**Discussion by archfrog** [community/community#127843] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

I'd like to suggest that you somehow indicate who the class of contributor in GitHub Issues.  This is for when you jump into an issue from an external link and quickly want to get an idea of who is saying what.

Normally, you'd see something like this:

```
foo commented on Mar 25
I blah blah blah.

bar commented on Mar 26
You can't blah blah without bluh bluh.

zoo commented on Mar 27
I disagree because blah blah.
```

I think it would help make the discussion clearer if each entry header indicated who the corresponding author was:

```
Guest foo commented on Mar 25
I blah blah blah.

Contributor bar commented on Mar 26
You can't blah blah without bluh bluh.

Owner zoo commented on Mar 27
I disagree because blah blah.
```

Basically, the intent is to ensure that readers can quickly see who is saying what.  If you jump into a random issue, not knowing the product at all, it is difficult to easily see who is an authority on the product and who is not.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).
